pkgs/applications/editors/emacs/elisp-packages/elpa-packages.nix

```diff
@@ -60,12 +60,6 @@ self: let
         else super.org-transclusion;
       rcirc-menu = markBroken super.rcirc-menu; # Missing file header
       cl-lib = null; # builtin
-      cl-print = null; # builtin
```

Comment:
Don't "clean up" old declarations. These are still useful for tools like https://github.com/nix-community/emacs-overlay?tab=readme-ov-file#extra-library-functionality and nix-doom-emacs.

Reply:
> These are still useful for tools like https://github.com/nix-community/emacs-overlay?tab=readme-ov-file#extra-library-functionality and nix-doom-emacs.

Could you elaborate why they are useful?

`rg` finds nothing for `tle` and `advice` in Nixpkgs, emacs-overlay and nix-doom-emacs.

While there are references for `cl-print`, they are all [guarded][1] by `? null`.  I am not sure how that `? null` is generated, so deleting `cl-print` may cause issues.

To sum up, I think it is safe to delete `tle` and `advice`.  I am also fine with keeping all three of them.  WDYT?

[1]: https://github.com/NixOS/nixpkgs/blob/ba195b10cc77767b2b2ac6e01fb7045f8d29f137/pkgs/applications/editors/emacs/elisp-packages/elpa-generated.nix#L1696

Reply:
That commit is removed and now all three of them remain unchanged.